pkg/sentry/csr/csr.go

```diff
@@ -160,7 +169,7 @@ func GenerateCSRCertificate(csr *x509.CertificateRequest, subject string, identi
 		cert.ExtraExtensions = append(cert.ExtraExtensions, pkix.Extension{
 			Id:       oidSubjectAlternativeName,
 			Value:    b,
-			Critical: false,
+			Critical: true,
```

Comment:
what is the impact of this change?

Reply:
It should be true to get this correctly. @yaron2 - comments?

Reply:
According to x509 and SPIFFE specs, a SubjAltName extension must be critical if subject name and DNS are not present.